Reregister testproj databases around test runs (#868)
github/vscode-codeql: extensions/ql-vscode/CHANGELOG.md

- Display CodeQL CLI version being downloaded during an upgrade. [#862](https://github.com/github/vscode-codeql/pull/862)
- Display a helpful message and link to documentation when a query produces no results. [#866](https://github.com/github/vscode-codeql/pull/866)

> - When running tests from the extension, first remove testproj databases affected by the tests and then try to reopen them after the tests complete. [#868](https://github.com/github/vscode-codeql/pull/868)

Reviewer: Would be better to phrase this as what the feature accomplishes, instead of how it was implemented.

```suggestion
- Test databases are automatically refreshed after a test run. [#868](https://github.com/github/vscode-codeql/pull/868)
```

Or something like that.
